src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -191,7 +191,7 @@ export function QueryPageContainerComponent(): JSX.Element {
 
             const allValuesText = t("Results.TableHead");
 
-            const colValues = generateTableHeaderValuesService(
+            const columnNames = generateTableHeaderValuesService(

```

Comment:
General consideration: Prefer your state variables over your temporary variables: So columnNames for state and e.g. colNames for the temp variable.
This is one more reason why i am no fan of the functional react components, in a class there would be a clear difference between state and local variable: state => this.columnNames, local => columnNames